src/ButtonGroup/index.js

```diff
@@ -0,0 +1,214 @@
+      field: { name, value },
+      form: { setFieldValue },
+    } = this.props;
+
+    const selectedCheckbox = e.target.id;
+    if (this.props.multiSelect) {
+      let selectedCheckboxes = Array.isArray(value) ? value.slice() : [];
+
+      if (
+        e.target.checked &&
+        selectedCheckboxes.indexOf(selectedCheckbox) === -1
+      ) {
+        selectedCheckboxes.push(selectedCheckbox);
+      } else if (!e.target.checked) {
+        selectedCheckboxes = without(selectedCheckboxes, selectedCheckbox);
+      }
+      return setFieldValue(name, selectedCheckboxes);
+    }
+
+    return setFieldValue(name, selectedCheckbox);
+  };
+
+  handleClick = (input) => {
+    document.getElementById(input).click();
+    let updatedState = this.state.selected.slice();
+
+    if (updatedState.includes(input) && this.props.multiSelect === true) {
+      updatedState = updatedState.filter((val) => {
+        return val !== input;
+      });
+    } else if (updatedState.includes(input)) {
+      return;
+    } else if (this.props.multiSelect) {
+      updatedState.push(input);
+    } else {
+      updatedState = [input];
+    }
+
+    this.setState({ selected: updatedState });
+  };
+
+  render() {
+    const {
+      label,
+      options,
+      classes,
+      field: { name },
+      multiSelect,
+    } = this.props;
+
+    const { selected } = this.state;
+    return (
+      <div className={classes.scaffold}>
+        {label && <Label>{label}</Label>}
+        <div className={classNames(classes.toggles, classes.inline)}>
+          {options.map((option) => (
+            <Fragment key={option.value}>
+              <label
+                className={classNames(
+                  classes.button,
+                  selected.includes(option.value) ? classes.buttonActive : null
+                )}
+                onClick={() => this.handleClick(option.value)}
+              >
+                <span
+                  className={
+                    selected.includes(option.value)
+                      ? classes.spanTextActive
+                      : classes.spanText
+                  }
+                >
+                  {option.label}
+                </span>
+              </label>
+              <input
+                id={option.value}
+                type={multiSelect ? "checkbox" : "radio"}
+                name={name}
+                className={classes.hiddenInput}
+                onChange={this.onChange}
+              />
+            </Fragment>
+          ))}
+        </div>
+      </div>
+    );
+  }
+}
+
+export default withStyles((theme) => ({
+  scaffold: {
+    margin: "8px auto 16px",
+  },
+  toggles: {
+    display: "flex",
+    flexDirection: "column",
+    flexWrap: "wrap",
+    alignItems: "flex-start",
+    marginLeft: "-12px",
+  },
+  inline: {
+    flexDirection: "row",
+  },
+  hiddenInput: {
+    display: "none",
+  },
+  button: {
+    background: "none",
+    border: `2px solid ${theme.colors["gray200"]}`,
```

Comment:
We need to figure something out to keep these matching the other components. (You'll notice that all the other form inputs are 1px border– the 2px border is something that was recently introduced on the Sites project redesign)

I don't have a great answer yet for how we should handle overrides from the theme– but let's definitely make this component match the rest then we can figure out overrides in a follow-up pass.